packages/@react-aria/slider/src/useSlider.ts

```diff
@@ -74,7 +75,8 @@ export function useSlider(
       currentPosition.current = null;
     },
     onMove({deltaX, deltaY}) {
-      let size = isVertical ? trackRef.current.offsetHeight : trackRef.current.offsetWidth;
+      let {height, width} = trackRef.current.getBoundingClientRect();
```

Comment:
for anyone else reviewing, offsetHeight does not exist on SVG's (it does it some browsers, but it shouldn't)